worlds/psychonauts/docs/setup_en.md

```diff
@@ -0,0 +1,48 @@
+# Psychonauts Archipelago Setup Guide
+
+<h2 style="text-transform:none";>Quick Links</h2>
+
+- [Game Info Page](../../../../games/Psychonauts/info/en)
+- [Player Options Page](../../../../games/Psychonauts/player-options)
+- [PsychoRando Setup Guide](https://docs.google.com/document/d/1b7QOnOLmTSvdC7A1YK3bsSmhtSOsAMs0XF5j-tyE6Zw/edit?usp=sharing)
+
+<h2 style="text-transform:none";>Required Software:</h2> <br>
+
+`Psychonauts` for PC
+
+- [Steam](https://store.steampowered.com/app/3830/Psychonauts/)
+- [Xbox](https://www.xbox.com/en-US/games/store/Psychonauts/C5HHPG1TXDNG)
+
+[Astralathe Mod Loader](https://gitlab.com/scrunguscrungus/astralathe/-/releases) <br>
+[PsychoRando AP Branch](https://github.com/Akashortstack/PsychoRando/releases) <br>
+[Psychonauts AP World](https://github.com/Akashortstack/Psychonauts_AP/releases) <br>
+
+<h3 style="text-transform:none";>Updating Your Host.yaml</h3>
+
+Find the folder that contains your Psychonauts and Astralathe installation, and copy the folder directory. For example,
+the most common place for the Steam version is "C:\\Program Files (x86)\\Steam\\steamapps\\common\\Psychonauts"
```

Comment:
Assuming the player has already installed Astralathe by this point, they should know where and how to access their Psychonauts installation. However, I've clarified the instructions for copying the file path since that's the focus. Further suggestions to clarify this still?

Reply:
Much clearer 👍